devices.js

```diff
@@ -7282,12 +7282,33 @@ const devices = [
         toZigbee: generic.light_onoff_brightness_colortemp.toZigbee.concat([tz.tint_scene]),
     },
 
-    // Salus
+    // Salus Controls
+    {
+        zigbeeModel: ['SPE600'],
+        model: 'SPE600',
+        vendor: 'Salus Controls',
+        description: 'Smart plug (EU socket)',
+        supports: 'on/off, power measurement',
+        fromZigbee: [fz.on_off, fz.SP600_power],
```

Comment:
Can you change `SP600_power` to `metering_power` and see if the power measurements are still correct? If so please use it instead.